exo-run/features/run.feature

```diff
@@ -54,3 +54,13 @@ Feature: running Exosphere applications
   Scenario: a service crashes during startup
     When trying to run the "crashing-service" application
     Then it prints "crasher exited with code 1" in the terminal
+
+
+  Scenario: hiding logs from certain service and dependency
+    Given a running "silenced-running" application
+    Then my machine is running the services:
+      | NAME  |
+      | web   |
+      | users |
```

Comment:
Let's list `exocom#{some-version}` here as well, since we're testing that it doesn't print logs for exocom.